velox/docs/monthly-updates/july-2022.rst

```diff
@@ -0,0 +1,58 @@
+****************
+July 2022 Update
+****************
+
+Documentation
+=============
+
+* Add documentation for Presto related :doc:`/functions/json`.
+
+Core Library
+============
+
+* Add `ApproxMostFrequentStreamSummary` that approximately computes the top frequent values from a large stream.
+* Add support for cast from a JSON type value to ARRAYS or MAPS of JSON type values.
+* Add Parquet writer implemented over Apache Arrow Parquet writer.
+* Add NegatedBytesRange filter.
+* Add PlanNode, PlanBuilder, registry support for Window operators.
```

Comment:
typo: operators -> operator

Perhaps, clarify a bit.

`* Add WindowNode to describe window functions computations. Window operator implementation is still in development.`